ngraph/CMakeLists.txt

```diff
@@ -341,7 +341,9 @@ if(NGRAPH_UNIT_TEST_ENABLE)
     set(BUILD_SHARED_LIBS OFF)
     include(cmake/external_gtest.cmake)
     if (NGRAPH_PDPD_FRONTEND_ENABLE)
-        include(cmake/external_npy.cmake)
+        if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/cnpy")
```

Comment:
not sure why this 'if exists' is here, but I see that it is used in 'IE Samples' in the same way, so ok.